testsuite/tests/typing-modes/portable-contend.ml

```diff
@@ -365,8 +365,6 @@ val foo : int @ shared -> unit = <fun>
 module Iarray = Stdlib_stable.Iarray
 
 let foo (r @ contended) = Iarray.get r 42
-(* CR zqian: The following should pass; the modal kind system should mode cross
```

Comment:
well, the issue here was that `r` is a `'a iarray`, which does not necessarily cross contention (since we don't know what `'a` is). I had removed the CR since this function should not in fact be accepted; instead now I've just split the test in two.